crates/compiler/parse/src/expr.rs

```diff
@@ -1044,6 +1134,7 @@ fn parse_defs_end<'a>(
     }
 }
 
+#[derive(Debug)]
```

Comment:
Wasn't sure about leaving this in. It was very helpful for debugging, but may have not been included for a reason.